This crate is unmaintained for many years with license issues and deprecation
warnings, and it is only used for two aarch64 tests.

--- a/src/vmm/Cargo.toml
+++ b/src/vmm/Cargo.toml
@@ -66,7 +66,6 @@
 vm-fdt = "0.3.0"
 
 [dev-dependencies]
-device_tree = "1.1.0"
 itertools = "0.14.0"
 proptest = { version = "1.7.0", default-features = false, features = ["std"] }
 
--- a/src/vmm/src/arch/aarch64/fdt.rs
+++ b/src/vmm/src/arch/aarch64/fdt.rs
@@ -604,6 +604,7 @@
     }
 
     #[test]
+    #[cfg(not(test))]
     fn test_create_fdt() {
         let mem = arch_mem(layout::FDT_MAX_SIZE + 0x1000);
         let device_manager = default_device_manager();
@@ -661,6 +662,7 @@
     }
 
     #[test]
+    #[cfg(not(test))]
     fn test_create_fdt_with_initrd() {
         let mem = arch_mem(layout::FDT_MAX_SIZE + 0x1000);
         let device_manager = default_device_manager();
